helm_deploy/values-dev.yaml

```diff
@@ -23,6 +23,7 @@ generic-service:
     PRISON_REGISTER_API_URL: "https://prison-register-dev.hmpps.service.justice.gov.uk"
     DPS_URL: "https://digital-dev.prison.service.justice.gov.uk/"
     SMS_NOTIFICATIONS_ENABLED: "true"
+    SYSTEM_CLIENT_REDIS_KEY: "bapv-staff-dev"
```

Comment:
I think this should just be `REDIS_KEY` here and in the other `values-*.yaml` files to match what's in `config.js`